src/main/java/de/hysky/skyblocker/skyblock/chat/ChatRulesConfigListWidget.java

```diff
@@ -0,0 +1,198 @@
+
+        @Override
+        public List<? extends Selectable> selectableChildren() {
+            return List.of();
+        }
+
+        @Override
+        public List<? extends Element> children() {
+            return List.of();
+        }
+
+        @Override
+        public void render(DrawContext context, int index, int y, int x, int entryWidth, int entryHeight, int mouseX, int mouseY, boolean hovered, float tickDelta) {
+            context.drawCenteredTextWithShadow(client.textRenderer, Text.translatable("text.autoconfig.skyblocker.option.messages.chatRules.screen.ruleName"), width / 2 - 125, y + 5, 0xFFFFFFFF);
+            context.drawCenteredTextWithShadow(client.textRenderer, Text.translatable("text.autoconfig.skyblocker.option.messages.chatRules.screen.ruleEnabled"), width / 2, y + 5, 0xFFFFFFFF);
+            context.drawCenteredTextWithShadow(client.textRenderer, Text.translatable("text.autoconfig.skyblocker.option.messages.chatRules.screen.modify"), width / 2 + 100, y + 5, 0xFFFFFFFF);
+        }
+    }
+
+    private class ChatRuleConfigEntry extends AbstractChatRuleEntry {
+        //data
+        private final int chatRuleIndex;
+        private final ChatRule chatRule;
+
+        private final List<? extends Element> children;
+
+        //widgets
+        private final ButtonWidget enabledButton;
+        private final ButtonWidget openConfigButton;
+        private final ButtonWidget deleteButton;
+
+        //text location
+        private final int nameX = width / 2 - 125;
+        //saved data
+        private double oldScrollAmount = 0;
+
+
+        public ChatRuleConfigEntry(int chatRuleIndex) {
+            this.chatRuleIndex = chatRuleIndex;
+            this.chatRule = ChatRulesHandler.chatRuleList.get(chatRuleIndex);
+
+            enabledButton = ButtonWidget.builder(enabledButtonText(), a -> toggleEnabled())
+                    .size(50, 20)
+                    .position(width / 2 - 25, 5)
+                    .build();
+
+            openConfigButton = ButtonWidget.builder(Text.translatable("text.autoconfig.skyblocker.option.messages.chatRules.screen.editRule"), a -> {
+                        client.setScreen(new ChatRuleConfigScreen(screen, chatRuleIndex));
+                    })
+                    .size(50, 20)
+                    .position(width / 2 + 45, 5)
+                    .tooltip(Tooltip.of(Text.translatable("text.autoconfig.skyblocker.option.messages.chatRules.screen.editRule.@Tooltip")))
+                    .build();
+
+            deleteButton = ButtonWidget.builder(Text.translatable("selectServer.delete"), a -> {
+                        oldScrollAmount = getScrollAmount();
+                        client.setScreen(new ConfirmScreen(this::deleteEntry, Text.translatable("skyblocker.shortcuts.deleteQuestion"), Text.translatable("skyblocker.shortcuts.deleteWarning", chatRule.getName()), Text.translatable("selectServer.deleteButton"), ScreenTexts.CANCEL));
```

Comment:
I would instead copy these translation strings and change the shortcut specific wording because it doesn't make sense